lib/plain/common-types.ts

```diff
@@ -383,6 +385,21 @@ export type PlainClientAPI = {
       params: OptionalDefaults<{ organizationId: string } & QueryParams>
     ): Promise<CollectionProp<UsageProps>>
   }
+  release: {
+    get(params: OptionalDefaults<GetReleaseParams>): Promise<ReleaseProps>
+    query(
+      params: OptionalDefaults<GetSpaceEnvironmentParams> & { query?: ReleaseQueryOptions }
+    ): Promise<CollectionProp<ReleaseProps>>
+    create(
+      params: OptionalDefaults<GetSpaceEnvironmentParams>,
+      data: ReleasePayload
+    ): Promise<ReleaseProps>
+    update(
+      params: OptionalDefaults<GetReleaseParams & { version: number }>,
+      data: ReleasePayload
+    ): Promise<ReleaseProps>
+    delete(params: OptionalDefaults<GetReleaseParams>): Promise<null>
```

Comment:
Would strongly recommend having the return being a `Promise<void>`

Reply:
Fixed 👍🏽